adoc/Private-Registry-Administration.adoc

```diff
@@ -743,3 +743,206 @@ suse-registry-harbor-registry-65854df7bc-mrfnj     2/2     Running   0
 suse-registry-harbor-trivy-0                       1/1     Running   1          24h
 suse-registry-harbor-trivy-1                       1/1     Running   0          24h
 ----
+
+=== Backup and Restore
+
+As an application that is deployed on kubernetes, backup and restore for {spr} can be performed by link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[Velero]. Velero is an application that performs backup/restore, disaster recovery, and migrate Kubernetes cluster resources and persistent volumes.
+
+Velero is able to fully cover the backup/restore for {spr} when it is deployed using internal services, such as database and redis. However, as Velero only covers kubernetes resources, when {spr} is deployed using external service providers, such as managed PostgreSQL database or object storage (Amazon S3/Azure Blob Storage), additional actions should be performed to ensure the backup of data external to the kubernetes cluster.
```

Comment:
```suggestion
Velero is responsible for the backup and restore for {spr} when deployed using internal services such as database and redis. However, as Velero is only responsible for Kubernetes resources. When {spr} is deployed using external service providers, such as managed PostgreSQL database or object storage (Amazon S3/Azure Blob Storage), additional actions need to be performed to ensure the backup of data external to the Kubernetes cluster.

Reply:
"internal services such as database and redis." - what is "internal services such as database" - like, a database? Or something more specific?

Reply:
Optionally SPR can deploy some required services (database and redis) inside the kubernetes cluster. The other option is to use an external database/redis and that is the case where velero is not able to backup the data from those services.